tap_github/comments.json

```diff
@@ -0,0 +1,98 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
```

Comment:
Can we not set `additionalProperties` here? We would prefer that the tap did not break if GitHub choose to send us more data.